opentreemap/treemap/models.py

```diff
@@ -953,6 +978,7 @@ def is_attr_set(attr):
         elif is_attr_set('tree'):
             self.map_feature = self.tree.plot
 
+        self.map_feature.update_updated_at()
```

Comment:
This is unnecessary, since the same call is made in `MapFeaturePhoto`, which is the superclass of `TreePhoto`.  Consider removing it (and possibly adding a comment to explain why it is unnecessary)